lib/module_importer.js

```diff
@@ -63,8 +63,9 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
     var relativePath = fragments.slice(1).join("/");
     var pkgRootDir = isRealFile ? importUtils.packageRootDir(path.dirname(prev)) : root;
     var jsFile = importUtils.getModuleByName(moduleName, pkgRootDir);
-    // includePaths here is a string, despite the config option being an Array
-    var includePaths = options.includePaths.split(path.delimiter);
+    // in some cases, includePaths here is a string, despite the config option being an Array
+    // other cases, it is an Array (or undefined if it was not passed as an option)
+    var includePaths = (options.includePaths && options.includePaths.split) ? options.includePaths.split(path.delimiter) : options.includePaths || [];;
```

Comment:
I’m still confused on why this is sometimes a string and sometimes an array. That seems like a core … wrongness… and by cleaning that up, we can write more concise and reliable code here.